.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "dashboard",
+  "name_pretty": "Monitoring Dashboards",
+  "product_documentation": "https://cloud.google.com/monitoring/docs",
+  "client_documentation": "https://googleapis.dev/nodejs/dashboard/latest",
+  "issue_tracker": "https://issuetracker.google.com/savedsearches/559785",
+  "release_level": "beta",
+  "language": "nodejs",
+  "repo": "googleapis/nodejs-monitoring-dashboards",
+  "distribution_name": "@google-cloud/monitoring-dashboards",
+  "api_id": "monitoring.googleapis.com",
```

Comment:
Eugh, this is probably going to create problems for drift tooling.  We should start a thread with martinv and colin.  We are making an assumption that every library has a one to one relationship to API_ID.